ci/jenkins_tests/run_multi_node_tests.sh

```diff
@@ -14,6 +14,11 @@ ROOT_DIR=$(cd "$(dirname "${BASH_SOURCE:-$0}")"; pwd)
 DOCKER_SHA=$($ROOT_DIR/../../build-docker.sh --output-sha --no-cache)
 echo "Using Docker image" $DOCKER_SHA
 
+######################## RAY BACKEND TESTS #################################
```

Comment:
I moved this to the front so I can see the Jenkins results more quickly, but will move back to the end before merging.

Reply:
On the other hand, it doesn't take very long I think, so maybe it's fine here.